docker-compose.yaml

```diff
@@ -16,7 +16,7 @@ services:
       - "./haproxy/maps:/etc/haproxy/maps:ro"
   postgrest-legacy-scripts:
     image: postgrest/postgrest
```

Comment:
this all looks great. do you think we should pin the postgrest versions, Frank? i am of two minds...

Reply:
Yea, it's a great question and I don't have an opinion really. Until we intentionally `docker pull` on this, it'll stay the same so I'm inclined to leave it, but I would also be just as fine pinning it. Like you -- two minds.